.githooks/pre-push

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env ruby
+
+puts "Attempting to push..."
```

Comment:
I was surprised by the output in my command line.

Maybe something more obvious:
```suggestion
puts "Running pre-push validation..."
```